src/compiler/parser.ts

```diff
@@ -1762,10 +1763,17 @@ namespace Parser {
         return hasJSDoc ? addJSDocComment(node) : node;
     }
 
+    const seeLink = /@(?:see|link)/;
+    function shouldParseJSDoc<T extends HasJSDoc>(node: T, comment: ts.CommentRange) {
+        if (!(globalThis as any).isTSC) return true;

```

Comment:
Is there a way to do this without smuggling in the boolean? How can we test that we don't break tsc?

Reply:
I mean, rather than a global context flag, best is probably to pass an option down into the parser from whatever's constructing the program, right? This way API consumers can intelligently select if _they_ need to skip jsdoc or not.